ambry-router/src/test/java/com.github.ambry.router/GetManagerTest.java

```diff
@@ -245,23 +246,35 @@ public void testFailureOnAllPollThatSends()
     setOperationParams(chunkSize);
     String blobId = router.putBlob(putBlobProperties, putUserMetadata, putChannel).get();
     mockSelectorState.set(MockSelectorState.ThrowExceptionOnSend);
+    FutureResult futureResult;
     try {
-      router.getBlobInfo(blobId).get();
+      futureResult = (FutureResult) router.getBlobInfo(blobId);
```

Comment:
I don't think you need the cast. `Future` supports `isDone()` - this is the only API you seem to use?